Add ability to disable the edition of configuration through API endpoints and UI (#7220)
wazuh/wazuh-documentation: source/user-manual/wazuh-dashboard/config-file.rst

| **Allowed values** | Any string                 |
+--------------------+----------------------------+

Configuration
-------------

configuration.ui_api_editable
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

> Define the ability to edit the configuration from UI or API endpoints. When disabled, this can only be edited from the configuration file, the related API endpoints are disabled, and the UI is inaccessible.

Reviewer: ```suggestion
Defines whether the configuration is editable via API endpoints and the UI. When disabled, editing is limited to the configuration file, rendering both the API endpoints and the UI inaccessible.
```
